earth_enterprise/rpms/opengee-server/post-install.sh

```diff
@@ -17,6 +17,12 @@
 set +x
 umask 002
 
+# we are not using this yet, but is probably good to have if we need to
```

Comment:
Minor: I don't think we should have code that's not used anywhere.  We can add it when we need it.

Reply:
I have killed all the post-install scripts in gradle now, because they only were involved in removing users and groups.  I could nuke the files as well...

Reply:
I think it's useful to print a message reminding a system administrator that they can remove the users and groups, if they want to.

Reply:
You have a good point.  I can re-activate the post-uninstalls and use it for that.